src/browser/components/MainPage.jsx

```diff
@@ -70,8 +70,8 @@ export default class MainPage extends React.Component {
             teamURL,
             teamIndex: index,
             originalURL: deeplinkURL,
-            url: `${teamURL.protocol}//${teamURL.host}${deeplinkURL.pathname}`,
-            path: deeplinkURL.pathname,
+            url: `${teamURL.protocol}//${teamURL.host}${deeplinkURL.pathname || ''}`,
+            path: deeplinkURL.pathname || '',
```

Comment:
if you use `URL` instead, pathname will always have the value of '/' as a minimum. and I know i've relied on that on the past. Until we unify use of url and URL, I'd advise to use '/' in case it is undefined.